resources/charts/stolon/templates/alerts-create-job.yaml

```diff
@@ -26,9 +27,11 @@ spec:
         - name: copy
           image: "{{ .Values.registry }}{{ .Values.image.repository }}:{{ .Values.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          # TODO(Sergei): delete `exit 0` after properly implementing Prometheus alert
           command: ["/usr/bin/dumb-init", "--", "/bin/bash", "-cex"]
           args:
             - |
+              /usr/bin/gravity resource rm alert stolon-replication-lag && exit 0 && \
```

Comment:
The intention is not to create the alerts if the alert `stolon-replication-lag` is removed

Reply:
I have no chance to rewrite stolon alert into prometheus. This one is just a temporary thing to fix the upgrade.

Reply:
ok, makes sense